### DISCUSSION TITLE
Lumen Support

### DISCUSSION THREAD

**Discussion by jsiebach** [worksome/envy#7] [Ideas]
Hey team, appreciate the work you put in on this project! It's a helpful tool for maintaining .env files.

I tried to install it on a Lumen app and it has just a couple of compatibility issues that would need to be resolved to support lumen.  Here's what I found:

1. use of `config_path()` in line 22 of `envy.php` config file. This could be replaced with `base_path('config')` I believe
2. use of blade facade in command output rendering.  Changing this to use `$this->table()` or a series of `$this->info()` calls to print the `$pendingUpdates` collection would solve this

Do you guys have any plans to provide Lumen compatibility?

Thanks!

**Comment by lukeraymonddowning** [comment]
Howdy! We have no plans to support Lumen at the moment. Although changes to add compatibility for Lumen are minor at the moment, we wouldn't want it to restrict the possibilities for Envy going forwards.

Really appreciate your ideas and input though!
